financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkstepupverification/LinkStepUpVerificationScreen.kt

```diff
@@ -199,6 +203,7 @@ internal fun LinkStepUpVerificationPreview(
     FinancialConnectionsPreview {
         LinkStepUpVerificationContent(
             state = state,
+            topAppBarState = TopAppBarState(hideStripeLogo = false),
```

Comment:
super nit - the stripe logo is just hidden in the consent screen, we can likely default this to false.

Reply:
I’d like to keep it as a required param. This change here is only necessary to keep the screenshots the same. Since the top app bar will be removed from panes in https://github.com/stripe/stripe-android/pull/8136, this code is actually going to be 🔥 soon.